doc/FAQ.md

```diff
@@ -34,13 +34,21 @@ distributed systems. I just wanted the fastest way to boot to Docker.
 
 **Persistent partition choice**
 
-Boot2Docker will first try to mount a partition labeled ``boot2docker-data``, if
+Boot2Docker will first try to mount a partition named ``boot2docker-data``, if
 that doesn't exist, it will pick the first ``ext4`` partition listed by ``blkid``.
 
+By default, the ISO image creates the ``boot2docker-vm`` partition, therefore no additional one is likely to be needed but you can still add some if you want to.
+
+Keep in mind that if you have 2 partitions with the same names but different labels,
+boot2docker will only see one of them. So, if you add a ``boot2docker-data`` partition, boot2docker will mount that one and forget ``boot2docker-vm``.
+However, if you name them differently, boot2docker will see all of them.
+E.g.: ``boot2docker-vm`` and ``partition2`` will both be mounted.
```

Comment:
From looking at the script in question (https://github.com/boot2docker/boot2docker/blob/175488c34ff9c05c1307705c47a4d8a1282951b8/rootfs/rootfs/etc/rc.d/automount#L3-L8), I don't see how it would ever even attempt to mount any partition that doesn't have a label of `boot2docker-data` explicitly.  Now, if you've got two partitions with that same label, it's true that only the first one will be mounted, but I don't see that really as a problem we can reasonably solve since we're mounting based on partition label.